addons/website/views/website_templates.xml

```diff
@@ -111,13 +111,16 @@
             </t>
         </t>
         <!-- `alternate`/`canonical` mainly useful to crawlers/bots/SEO tools, which test the website as public user -->
-        <t t-if="request and request.is_frontend_multilang and website and website.is_public_user()">
+        <t t-if="request and request.is_frontend_multilang">
             <t t-set="alternate_languages" t-value="website._get_alternate_languages(canonical_params=canonical_params)"/>
-            <t t-foreach="alternate_languages" t-as="lg">
-        <link rel="alternate" t-att-hreflang="lg['hreflang']" t-att-href="lg['href']"/>
-            </t>
+            <t t-foreach="alternate_languages" t-as="lg" t-if="len(alternate_languages) > 1">
+        <link rel="alternate" t-att-hreflang="lg['url_code']" t-att-href="lg['href']"/>
```

Comment:
Why renaming `hreflang` into `url_code` which now sounds like it's the `url_code` properties while it is not?
Hm.. Checking deeper it seems like it is now the `url_code` which is not correct, it shouldn't be, `url_code` is an input that the user can customize to his will, we can't be sure it will respect the expected format.

Before:
```xml
<link rel="alternate" hreflang="fr-be" href="http://127.0.0.1:8069/fr_BE/jobs/detail/developpeur-experimente-4"/```xml
<link rel="alternate" hreflang="fr-fr" href="http://127.0.0.1:8069/francais/jobs/detail/developpeur-experimente-4"/>
```
After:
```xml
<link rel="alternate" hreflang="fr_BE" href="http://127.0.0.1:8069/fr_BE/jobs/detail/developpeur-experimente-4"/>
<link rel="alternate" hreflang="francais" href="http://127.0.0.1:8069/francais/jobs/detail/developpeur-experimente-4"/>
```

Also, on top of that, it seems like another behavior was lost, if you had only one french lang, it would use `fr` without specifying the country code:
Before:
```xml
<link rel="alternate" hreflang="fr" href="http://127.0.0.1:8069/fr_BE/jobs/detail/developpeur-experimente-4"/```xml
```
After:
```xml
<link rel="alternate" hreflang="fr_BE" href="http://127.0.0.1:8069/fr_BE/jobs/detail/developpeur-experimente-4"/>
```

Reply:
thanks, fixed.